config-templates/ibm/ess/1.1/custom-parameters.json

```diff
@@ -0,0 +1,93 @@
+[
+	{
+		"description": "Mount path of the primary file system",
+		"displayname": "Scale Host Path",
+		"name": "scale-host-path",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "Cluster ID of the pirimary IBM Spectrum Scale Cluster",
+		"displayname": "Cluster ID",
+		"name": "cluster-id",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "Primary file system name",
+		"displayname": "Primary FS",
+		"name": "primary-fs",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "FQDN or IP address of the GUI node of scale cluster that is specified against the Cluster ID parameter",
+		"displayname": "GUI Host",
+		"name": "gui-host",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "Name of secret for username password to connect to Primary Cluster GUI",
+		"displayname": "Primary Cluster, Secret Name",
+		"name": "secret-name",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "Username to connect to Primary Cluster GUI",
+		"displayname": "Primary Cluster GUI Username",
+		"name": "gui-api-user",
+		"required": "true",
+		"type": "base64"
+	},
+	{
+		"description": "Password to connect to Primary Cluster GUI",
+		"displayname": "Primary Cluster GUI Password",
+		"name": "gui-api-password",
+		"required": "true",
+		"type": "base64"
+	},
+	{
+		"description": "node1-k8-ip",
```

Comment:
Can we take the descriptions from the parameter reference table for these?

Reply:
Will update descriptions.  Was awaiting feedback from development on whether these fields were necessary.